src/mixins/filing-template-mixin.ts

```diff
@@ -109,7 +120,7 @@ export default class FilingTemplateMixin extends Vue {
         },
         parties,
         shareStructure: {
-          shareClasses: this.getShareClasses
+          shareClasses: shareStructure
```

Comment:
Or you could name it the same and use the object property value shorthand.
https://alligator.io/js/object-property-shorthand-es6/

Reply:
Great idea! Updated in next commit